munch-match/src/components/resultsLayout.jsx

```diff
@@ -1,105 +1,157 @@
             });
-            this.initializeMap(locations); 
-          })
-          .catch(error => {
-            console.error('Error fetching results:', error);
-          });
-      }
-      // Function to initialize embedded map
-      initializeMap = (foodLocations) => {
+    }
+    // Function to initialize embedded map
+    initializeMap = (foodLocations) => {
         const center = {lat: foodLocations[0].latitude, lng: foodLocations[0].longitude}
         const map = new google.maps.Map(document.getElementById('map'), {
-          zoom: 13, 
-          center: center
+            zoom: 13,
+            center: center
         });
 
         // Loop through each location and extract its coordinates and make a new marker on the map
         foodLocations.forEach(location => {
             new google.maps.Marker({
-            position: {lat: location.latitude, lng: location.longitude}, 
-            map: map
-          })
+                position: {lat: location.latitude, lng: location.longitude},
+                map: map
+            })
         })
-      }
-
-    render(){
-        console.log(this.selectedType)
-        const { results } = this.state;
-        console.log(results[2]);
-
-        return(
-        //To export multiple components, surround it with a <div> tag
-        <div className={styles.resultsContainer}>
-            <div className={styles.resultsRow}>
-                <ResultCard result={results[0]}/>
-                <ResultCard result={results[1]}/>
-                <ResultCard result={results[2]}/>
-            </div>
-            <div className={styles.resultsRow}>
-                <ResultCard result={results[3]}/>
-                <ResultCard result={results[4]}/>
-                <ResultCard result={results[5]}/>
-            </div>
-            <div id="map" className={styles.map}></div>
-            <div>
-            <button className={styles.buttonResultLayout} onClick={() => this.onBackClick()} style={{width: "30%"}}><h3>Choose Again</h3></button>
+    }
+
+    updateRating = (range) => {
+        this.setState({ratingRange: range});
+        this.componentDidMount();
+    }
+
+    updatePriceRange = (price) => {
+        const newPriceRanges = { ...this.state.priceRanges };
+        newPriceRanges[price] = !newPriceRanges[price];
+        this.setState({ priceRanges: newPriceRanges });
+        this.componentDidMount();
+    }
+
+    render() {
+        const {results, ratingRange} = this.state;
+
+        return (
+            //To export multiple components, surround it with a <div> tag
+            <div className={styles.resultsContainer}>
+                <h3>Rating Range</h3>
+                <RangeSlider min={0} max={5} defaultValue={[0, 5]} step={0.1}
+                             onInput={(range) => this.updateRating(range)}/>
+                <p>{ratingRange[0]} - {ratingRange[1]} stars</p>
+
+                <h3>Price Range</h3>
+                <div style={{display: "inline-flex"}}>
```

Comment:
really minor, but could consider moving this into the CSS module instead of adding the CSS style here - it keeps it consistent with the other code in this section.
So perhaps something like this 
<div className={styles.inlineFlexDiv}>